webapp/app/routes/app/proposals/$id/_route.tsx

```diff
@@ -444,10 +452,15 @@ function StatusIndicator({
   );
 }
 
-function displayProposalState(state: PROPOSAL_STATES) {
+function displayProposalState(
+  state: PROPOSAL_STATES,
+  times: { totalDays: number; currentDay: number } | null
+) {
   let color: string;
   let label: string;
 
+  const timeData = times ? `(day ${times.currentDay} out of ${times.totalDays})` : "";
```

Comment:
Pending Tomi's opinion, consider changing this to `( Day <current> of <total> )`
for cleaner look

Reply:
I also belive it looks nicer. I applied this.